### PR TITLE
problems: warn for integer named testcases

### PR DESCRIPTION
**Pull request by remi-dupre**
Using an integer as a name for a testcase can lead to unexpected behaviors while parsing problem.prop.
Thus, is should be unrecommended as long as the parser tries to cast values to integers.

### CODE REVIEW

File: prologin/problems/views.py [hunk 1]
```diff
@@ -210,6 +211,25 @@ def get_context_data(self, **kwargs):
         context['challenge'] = challenge
         context['templatable_languages'] = list(problem.language_templates.keys())
 
+        # As an integer name can be missinterpreted by the parser in the test
+        #   case section, a warning is displayed to staff members if there is
```
Comment (zopieux) [inline prologin/problems/views.py:215]: Please don't indent comments.

File: prologin/problems/views.py [hunk 2]
```diff
@@ -210,6 +211,25 @@ def get_context_data(self, **kwargs):
...
+            for test in problem.tests:
+                if test.name.isdigit():
+                    integer_names.append(test.name)
+
+            if integer_names:
+                names_str = ', '.join(integer_names)
+                messages.add_message(
+                    self.request, messages.WARNING,
+                    'Some testcase names are integers. This may be an issue'
+                    ' while parsing <em>problem.props</em>, so you should'
```
Comment (zopieux) [inline prologin/problems/views.py:229]: `<code>`

**Review by zopieux — CHANGES_REQUESTED** [review]
I'm not convinced that problem validation should go in a random Django web view. This is `problems/`'s concern. It could be okay to put that in #67, which has the explicit goal of finding random stupid/inconsistencies across our data models. But then again, this should end up in `problems/`.

**Comment by remi-dupre** [comment]
Not really usefull

**Pull request closed without merging**